qln/routing.go

```diff
@@ -576,7 +576,11 @@ func (nd *LitNode) PopulateRates() error {
 
 	jsonFile, err := os.Open(ratesPath)
 	if err != nil {
-		return err
+		if !os.IsNotExist(err) {
+			return err
+		}
+		logging.Infof("Rates file not found.")
+		return nil
```

Comment:
We must return errors to the handler above, so something like return `return fmt.Errorf("")` should be fine. The info log helps as well, although, it could be marked `Debug` (since we have tons of `Info` logs)

Reply:
Not finding the rates file isn't an error.  Although I guess I should put that `!os.IsNotExist(err)` in the caller.